test/benchmarks/netperf/Makefile

```diff
@@ -0,0 +1,64 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+all: docker nptests push docker-client
+
+# Unfortunately, we need to build two Docker containers - one with no ports exposed
```

Comment:
Ouch, I wasn't aware of that - I will eliminate a lot of parallel references to the two images.